### PR TITLE
updated parameters for allele schema

### PR DESCRIPTION
**Pull request by mattheww95**
Updated allele calling schema options.

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -126,6 +119,25 @@
...
         },
+        "allele_schema_options": {
+            "title": "Allele Schema Options",
+            "type": "object",
+            "description": "Specify an allele calling schema.",
+            "default": "",
+            "properties": {
+                "force_allele_scheme": {
+                    "type": "string",
+                    "description": "Specify the path to an allele calling schema to be used. This schema will be used for allele calling of all samples."
```
Comment (apetkau) [inline nextflow_schema.json:130]: Could you include a regex `pattern` here like other path parameters and set the format to `directory-path`?

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks great. Thanks so much Matthew 😄 .  A few comments.

File: nextflow_schema.json (hunk 1 above)
Comment (mattheww95) [reply nextflow_schema.json]: fixed in: https://github.com/phac-nml/mikrokondo/pull/104/commits/0c74a1fc4dfd86f309bd2c2202662e8ec0e3cec9

File: nextflow_schema.json [hunk 2]
```diff
@@ -126,6 +119,25 @@
             ],
             "description": "The location of databases used by mikrokondo"
         },
+        "allele_schema_options": {
+            "title": "Allele Schema Options",
+            "type": "object",
+            "description": "Specify an allele calling schema.",
```
[the remark this answers is not in the document]
Comment (mattheww95) [reply nextflow_schema.json:125]: fixed in: https://github.com/phac-nml/mikrokondo/pull/104/commits/0c74a1fc4dfd86f309bd2c2202662e8ec0e3cec9

File: nextflow_schema.json [hunk 3]
```diff
@@ -126,6 +119,25 @@
...
+            "default": "",
+            "properties": {
+                "force_allele_scheme": {
+                    "type": "string",
+                    "description": "Specify the path to an allele calling schema to be used. This schema will be used for allele calling of all samples."
+                },
+                "lx_allele_database": {
```
[the remark this answers is not in the document]
Comment (mattheww95) [reply nextflow_schema.json:128]: Good call! 

fixed in: https://github.com/phac-nml/mikrokondo/pull/104/commits/0c74a1fc4dfd86f309bd2c2202662e8ec0e3cec9

**Review by apetkau — APPROVED** [review]
Looks good to me. Thanks so much 😄

**Pull request merged**